llvm/test/CodeGen/AArch64/PNRtoPPRCopy.mir

```diff
@@ -23,8 +11,8 @@ frameInfo:
 machineFunctionInfo:
   hasRedZone:      false
 body:             |
-  bb.0 (%ir-block.0):
-    ; CHECK-LABEL: name: ptrue_d
+  bb.0:
+    ; CHECK-LABEL: name: pnr_to_ppr
     ; CHECK: renamable $pn8 = PTRUE_C_D
     ; CHECK-NEXT: $p0 = ORR_PPzPP $p8, $p8, killed $p8, implicit-def $pn0
```

Comment:
Should it do an `implicit-def` of `$pn0` here? It's copying `$pn8` -> `$p0`.
I would expect there only to be an `implicit-def` for a `$pn*` register if the destination of the COPY is a predicate-as-counter register.